app/src/main/res/layout-land/selection_interaction_item.xml

```diff
@@ -14,17 +13,17 @@
     android:id="@+id/interaction_container_linear_layout"
     android:layout_width="match_parent"
     android:layout_height="wrap_content"
-    android:layout_marginStart="28dp"
-    android:layout_marginTop="@dimen/divider_margin_top"
-    android:layout_marginEnd="28dp"
-    android:layout_marginBottom="@dimen/divider_margin_bottom"
+    android:layout_marginTop="24dp"
+    android:layout_marginBottom="0dp"
```

Comment:
Should we skip specifying 0dp or will it be good for reference?

Reply:
I actually kept it intentionally because margins are relative, for View1 it can be marginBottom but for View2 it can be marginTop, so to avoid confusion, I have mentioned `marginBottom=0dp` in almost every possible scenario.

Reply:
Okay